packages/docusaurus-theme-classic/src/theme/Footer/index.tsx

```diff
@@ -106,10 +106,9 @@ function SimpleLinks({links}: {links: SimpleFooter['links']}) {
   return (
     <div className="footer__links">
       {links.map((item, key) => (
-        <>
+        <React.Fragment key={key}>
```

Comment:
Looks like we could prevent this with ESLint rule (disabled by default atm):

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md#checkfragmentshorthand-default-false

Reply:
Interestingly, this rule is disabled in airbnb because of "false positives". I think we can enable some React rules ourselves

Reply:
honnestly I never liked eslint airbnb, and many people have the same feeling

In the long term we should remove it imho

Reply:
I like it as a solution that just works... We aren't tied to it (while many others are forced to have a style guide and stick to it) so we can tweak it however we want